compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/DesktopPopup.desktop.kt

```diff
@@ -185,17 +185,15 @@ private fun PopupLayout(
                     val width = constraints.maxWidth
                     val height = constraints.maxHeight
 
-                    val windowSize = IntSize(
-                        width = width,
-                        height = height
-                    )
+                    val windowRect = IntRect(0, 0, width, height)
+                    val anchorRect = if (parentBounds == IntRect.Zero) windowRect else parentBounds
```

Comment:
We should always choose `windowRect`, not parentBounds. AlertDialog shouldn't be aligned in the parent container.

This code isn't works as expected:
```
import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.padding
import androidx.compose.material.AlertDialog
import androidx.compose.material.Button
import androidx.compose.material.ExperimentalMaterialApi
import androidx.compose.material.Text
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.dp
import androidx.compose.ui.window.singleWindowApplication

@OptIn(ExperimentalMaterialApi::class)
fun main() = singleWindowApplication {
    Box(Modifier.padding(start = 400.dp)) {
        AlertDialog(
            onDismissRequest = {},
            title = { Text("Title") },
            text = { Text("Text") },
            confirmButton = {
                Button(
                    onClick = {}
                ) {
                    Text("Confirm")
                }
            },
            dismissButton = {
                Button(
                    onClick = {}
                ) {
                    Text("Dismiss")
                }
            }
        )
    }
}
```
![image](https://user-images.githubusercontent.com/5963351/141104427-0026fa82-27c2-41b8-9bc3-5f8af108ade1.png)

Reply:
Done.